### PR TITLE
adds gcp.analytics.cloud-task (#308) - closes #308

### LINKED ISSUE #308: CloudTask support in GCP
cloud task is vastly used push delivery targetted for one delivery, with support for retry and concurrency.

### PR DESCRIPTION
**Pull request by abdulkaderjeelani**
PR for Issue #308

### CODE REVIEW

File: docs/nodes/gcp.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Node classes list of gcp provider.
 ## gcp.analytics
 
 - **diagrams.gcp.analytics.Bigquery**, **BigQuery** (alias)
+- **diagrams.gcp.analytics.CloudTask**
```
Comment (ryanyuan) [inline docs/nodes/gcp.md:11]: This has been implemented in diagrams.gcp.devtools.Tasks.

Comment (abdulkaderjeelani) [reply docs/nodes/gcp.md:11]: Hi ryanyuan, Can we change the name space, devtools is quite misleading. I also tried to locate a separate namespace  for messaging (pub sub, cloud tasks), only found analytics. Any thoughts ?

Comment (ryanyuan) [reply docs/nodes/gcp.md:11]: If you look at [https://cloud.google.com/products#section-9](https://cloud.google.com/products#section-9), Cloud Task is actually under dev tool category. Plus if you could open the GCP console, you can find the same categorisation on the navigation menu. And to be honest, Cloud Task is not for analytics. 

The namespace under gcp are based on the official categorisations from the link above.

Comment (mingrammer) [reply docs/nodes/gcp.md:11]: As @ryanyuan said, Cloud Tasks is under the dev tool category officially. Change the namespace is not a good option I think.

**Pull request closed without merging**